fs/nfsd/vfs.c

```diff
@@ -127,11 +164,21 @@ nfsd_cross_mnt(struct svc_rqst *rqstp, struct dentry **dpp,
 	struct path path = {.mnt = mntget(exp->ex_path.mnt),
 			    .dentry = dget(dentry)};
 	unsigned int follow_flags = 0;
+	int is_snapdir = 0;
```

Comment:
Will things barf if we don't also `#if CONFIG_TRUENAS` this?  (Barf when off, so meh?)

Reply:
Right